flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/DefaultConfigurableOptionsFactory.java

```diff
@@ -156,6 +160,24 @@ public ColumnFamilyOptions createColumnOptions(
             blockBasedTableConfig.setBlockCacheSize(getBlockCacheSize());
         }
 
+        if (isOptionConfigured(USE_BLOOM_FILTER)) {
+            final boolean enabled = Boolean.parseBoolean(getInternal(USE_BLOOM_FILTER.key()));
+            if (enabled) {
+                final double bitsPerKey =
+                        isOptionConfigured(BLOOM_FILTER_BITS_PER_KEY)
+                                ? Double.parseDouble(getInternal(BLOOM_FILTER_BITS_PER_KEY.key()))
+                                : BLOOM_FILTER_BITS_PER_KEY.defaultValue();
+                final boolean useFullFilter =
+                        isOptionConfigured(BLOOM_FILTER_USE_FULL_FILTER)
+                                ? Boolean.parseBoolean(
+                                        getInternal(BLOOM_FILTER_USE_FULL_FILTER.key()))
+                                : BLOOM_FILTER_USE_FULL_FILTER.defaultValue();
+                BloomFilter bloomFilter = new BloomFilter(bitsPerKey, useFullFilter);
```

Comment:
I'm afraid that you misundstand the parameters of `BloomFilter`. The 2nd parameter means whether is block-based filter. If it sets as true, it will generate block-based filter instead of full filter.

Reply:
You're right... I don't why I mix them mistakenly yesterday :(.  

To align with the method `public BloomFilter(final double bitsPerKey, final boolean useBlockBasedMode)` in RocksDB, I'll use block-based option instead of full filter option here,